gen/build_deploy/bash.py

```diff
@@ -397,12 +397,12 @@ def validate_custom_check_bins_dir(custom_check_bins_dir):
     while [[ ! -d "$DIRNAME" ]]; do
         DIRNAME="$(dirname "$DIRNAME")"
     done
-    read -r filesystem_device filesystem_type <<<"$(df --portability --print-type "$DIRNAME" | awk 'END{print $1,$2}')"
+    read -r filesystem_device filesystem_type filesystem_mount <<<"$(df --portability --print-type "$DIRNAME" | awk 'END{print $1,$2,$7}')"
```

Comment:
Why are you not passing `$DIRNAME` to `xfs_info` directly so we don't need to parse the root dir here.

Reply:
The `df` output gives always the **mount point**, while the `DIRNAME` input could, in theory, happen to be a regular directory depending on the input.

Reply:
Yeah, and the manpage says `The mount-point argument is the pathname of the directory where the filesystem is mounted.` so technically we wouldn't need to parse the mount point but just pass in a directory that's somewhere under that mount point. That's why I was asking whether there's a specific reasoning behind parsing the actual mount point from `df`'s output. But you're right that passing the mount point is a bit cleaner so let's do that.

Reply:
Sorry, one more thing: CI is complaining that the line is too long now.
```suggestion
    read -r filesystem_device filesystem_type filesystem_mount <<<"$(df --portability \
        --print-type "$DIRNAME" | awk 'END{print $1,$2,$7}')"
```